ev3dev/motor.py

```diff
@@ -80,7 +80,10 @@ def __str__(self):
         return ("%d rps" % self)
 
     def get_speed_pct(self, motor):
-        return motor.rps_to_speed(self)
+        """
+        Return the motor speed percentage to achieve desired rotations-per-second
+        """
+        return float(self/motor.max_rps) * 100
```

Comment:
Side-note: does this actually require the `float` conversion? As of Python 3 it'll do floating-point division anyway.

Reply:
I still do this out of python2 habit...will drop the float call